src/sql/workbench/parts/charts/browser/imageInsight.ts

```diff
@@ -57,6 +58,10 @@ export class ImageInsight implements IInsight {
3MbFsv4JKkW3gFzk26Dc6EWy/gqqRbeAxOTLqNz5xbL+DCpFu4B85Kuk3BtFsv4NqkW/gJTkm6zcLMWy/g8qRb+AbnI90mYvKtF3CF0i2sdD7zb72Ai5Rua4MzkW7TscTWC7hO6bYqOA3pNiOrbL2AS5Vu64FzkG6TstDWC7ha6bYSOAHpNi9rbb2AC5ZuC4AXrzpPzXJbL+CapdvU4JWrzrOz4tYLuGzpNil42arzAiy69QKuXLpNB15Tui3Dulsv4O6l20TgBaXbSiy99QIWUHUeH7yXdFuM1bdewA6qziODN5Ju65Gt/wFrqDqPCd5Fui1Jtm6wiarzaOAtpNuqZOs/wDik2zjg+aXbwmTrBBORbiOAJ5dua5Ot3wFDkW59g2eWbsuTrd8Hc5FuvYKnlW4hW38CRiPd+gPPKd3CF9n6MzAd6dYTeELpFr7J1l+AAUm3PsCzSbdwQ7b+GsxIul0Nnkq6hZ9k62+BMUm368DzSLfwi2z9XTAp6XYFeBLpFu6RrW8Aw5Ju54JnkG7hAdn6NjAv6XYW+HXpFh6TrW8GI5Nu7cHvSrfwlGz9EzA16dYS/KJ0C6/I1j8Eg5NubcBvSbfwBtn652B20u1o8CvSLbxHtr4LjE+6HQe+X7qFt8nW94IJSrcjwDdLt7CFbP0AMETptg98p3QLG8nWjwFzlG6fgm+TbmE72fphYJRV543gS6Rb+Ihs/Ugwzarze+Bvq87hU7L1g8FAq86vwF9VncMOsvXjwUyrzo/B56vOYR/ZeiuwV+l2D3yy6hx2k603BKuVbj/BZ6RbOIhsvS2Yr3T7BlW6hePI1puDEUu3DP1EsvUzwJTlk38PLcjWTwKDfqQ/HRqQrZ8HZv1bfy60IVs/FYz7Vn8iNCNbPxtMXLqFlmTrp4KJ3+pPhGZk6+eBcf/WnwttyNZPArN+pD8dGpCtnwEGLZ/8e2hBtt4cTFm6Ze4nkq23BSOWbt+gSrdwHNl6QzBf6fYTfEa6hYPI1luB4Uq3e+CT0i0cQbbeBExWuj0Gn5duYTfZ+vFgrNLtFfgr6Rb2ka0fDGYq3d4Dfyvdwg6y9SPBQKXbFvANVefwEdn6YWCX0m07+J6qc9hOtn4MWKR0+xR8W9U5bCRbPwBsUbrtA99ZdQ5byNb3ghVKt4PAl0u38DbZ+i6wP+l2KPgJ6RbeI1v/HCxPujUAPyTdwhtk65+AwVWdm4Gfk27hFdn6ZjC1qnNj8KPSLTwlW98GRlZ1PgX8tHQLj8nWN4B5VZ1PBA8g3cIDsvV3wbCk2xXgSaRbuEe2/haYlHS7DjyPdAu/yNZfgzFJt6vBU0m38JNs/QWYkXTrAzybdAs3ZOvPwICkW0/gCaVb+CZbfwimI936A88p3cIX2fp9MBrp1it4WukWsvW7YC7SrW/wzNJtebJ1gqFItxHAk0u3tcnWf4CJSLdxwPNLt4XJ1v8D45Buo4G3kG6rkq0bzEK6jQneRbotSbb+BwxCuo0M3ki6rUe2Pvkg8F5V55VYfetYgHSbBbxd1XkZlt467l66zQXeseq8ButuHbcu3WYEb1p1XoBFt477lm5Tg1eWbrOz4tZx09JtAfDi0m1qlts67li6LQNeX7rNy1pbx+1Kt8XAIUi3SVlo67hX6bYkOArpNiOrbB03Kt0WBgci3aZjia3jLqXb8uBYpNtczL913KJ0C1/gcKTbREy+ddyfdAs34Iik2yzMvHXcnHQLv8BBSbcpmHPruLCqc3gAjku6jc+EW8dVVZ3DU3Bo0m1wZts6LqnqHN4ARyfdRmaqreN6pFvYAs5Qug3LPFvHxUi3sB2cpHQbk0m2jiuRbuFTcJ7SbUBm2DouQ7qFfeBUpdtoDL91XIN0C0eAs5VuQzH21nEB0i0cB05Yuo3DwFvH0Uu3cDQ4Z+k2CKNuHYcu3UIbcNrSbQSG3DqOW7qFluDMq859M97WccrSLbQHJ1917pjBto7zlW7hLHD+VedeGWnrOFnpFk4HFyHdumSYreNMpVu4CFyHdOuPMbaO05Ru4VJwKdKtMwbYOs5RuoUOwNVIt57ofes4QekWugEXJN26oeut4+ykW+gMXJN064N+t45Tk26hS3BZ0q0DOt06zku6hY7BlUm3q+lx6zgp6Ra6Bxcn3S6lu63jjKRbGARcn3S7jr62jtORbmEocInS7SI62jrORbqFAcFVSrcr6GXrOBHpFoYFFyrdTqeLreMspFsYHFyrdDuX67eOU5BuYQpwudLtRC7eOt5fuoWJwBVLt7O4bOt47apzmA5ctHQ7hWu2jheuOodJwXVLt/ZcsHW8atU5TA0uXbo15uyt4yWrzmEBcPXSrSWnbh2vJ93CSmAD0q0Z520dLybdwnpgCVXnBpy0dbyPdAurgj1UnY/mjK3jTaRbWBusoup8KM23jneQbiF8gXlIt+Nou3U8vXQL4QaMRLodRMOt47mlWwi/wFSk2xG02jqeWLqF8AAMRrrtpsnW8azSLYSnYDbSbR/Hbx1PKd1CeAOMR7rt4OCt4/mkWwhvgwlJt085cut4MukWwkYwJOn2EYdtHc8k3UL4CMxJum3nmK3jaaRbCDvAqKTbRg7YOp5DuoWwG0xLum1h79bxBNIthIPAwKTb2+zaOn5buoVwKJiZdHuPz7eOX5VuITQAY5Nub/Dh1vF70i2EZmBy0u0Vn2wdvyTdQmgMhifdnrJ56/gN6RbCKWB+0u0x27aOb5duIZwIRijdHrBh6/he6RbC6WCK0u0e724d3yjdQrgIDFK6/eKtreO7pFsIl4JZSrefvN46vkW6hdABGGfV+ZtnW8dfVp1D6AZMtOr8xcOt42+qziF0BoZadX60dXy66hxCr2CxUunO1vG5qnMIfYPdyvLv3Do+Id1CGAQMWP7YOpp0C2EoMOPif1tHkG4hDAjG7K3jX6VSCONyu+c/W7/9/6o+GsLo1En/Vf/rVn8qhCn4s+q///k/Gmpj6opufUIAAAAASUVORK5CYII=`;
```

Comment:
We probably don't want to include an image in the source like this, include it as a file and use it in the code instead.